src/App.js

```diff
@@ -1,27 +1,26 @@
-import "./App.css";
-import logo from "./logo.png";
-import {Routes,Route} from "react-router-dom";
-import Home from "./Pages/Home/Home";
-import Product from "./Pages/Product/Product";
-import Wishlist from "./Pages/Wishlist/Wishlist";
-import Cart from "./Pages/Cart/Cart";
-import Login from "./Pages/Auth/Login";
-import Signup from "./Pages/Auth/Signup";
-import Mockman from "mockman-js"
-import Forgot from "./Pages/Auth/forgot";
+import './App.css';
+import logo from './logo.png';
+import { Routes, Route } from 'react-router-dom';
+import Home from './Pages/Home/Home';
+import Product from './Pages/Product/Product';
+import Wishlist from './Pages/Wishlist/Wishlist';
+import Cart from './Pages/Cart/Cart';
+import Login from './Pages/Auth/Login';
+import Signup from './Pages/Auth/Signup';
+import Mockman from 'mockman-js';
+
 function App() {
   return (
-    <div className="App">
-     <Routes>
-       <Route path="/" element={<Home/>}></Route>
-       <Route path="/product" element={<Product/>}></Route>
-       <Route path="/wishlist" element={<Wishlist/>}></Route>
-       <Route path="/cart" element={<Cart/>}></Route>
-       <Route path="/login" element={<Login/>}></Route>
-       <Route path="/signup" element={<Signup/>}></Route>
-       <Route path="/forgot" element={<Forgot/>}></Route>
-       <Route path="/mock" element={<Mockman/>}></Route>
-     </Routes>
+    <div className='App'>
+      <Routes>
+        <Route path='/' element={<Home />}></Route>
+        <Route path='/product' element={<Product />}></Route>
+        <Route path='/wishlist' element={<Wishlist />}></Route>
```

Comment:
These should be private routes?

Reply:
For a time being I have left it. I will add route to every project.